airflow/providers/amazon/provider.yaml

```
@@ -103,6 +103,7 @@ dependencies:
   # We should investigate if we can upgrade to a newer version of lxml and xmlsec
   # Tracked in https://github.com/apache/airflow/issues/39103
   - xmlsec<1.3.14
+  - jmsepath
```

```suggestion
  - jmespath
```

I guess this just a typo

---

Serves me right for not copy/pasting, especially since I always transpose those characters for this one haha.